### ISSUE TITLE
link is not a browser URL for SSH remote URLs

### ISSUE THREAD

**Issue by kevincantu** [calebmeyer/git-links#2]
Your fixes after #1 got me back in business, @calebmeyer, thanks!  :smiley_cat: 

Now, however, I've run into this.  If I have a repository whose `origin` remote is like so:
```
>git remote -v
origin  git@github.com:sample/sample.git (fetch)
origin  git@github.com:sample/sample.git (push)
```
Then the `git config --get remote.origin.url` command will print out:
```
git@github.com:sample/sample.git
```
And git-links will give me URLs like so:
```
git@github.com:sample/sample/blob/6500a16c43c461ad5bb43d907a7e52c4fd13a2bf/path/to/file.m#L9
```

It would be useful (whether you want this in another function, or what) to have a GitHub (or whatever other service) URL like so, instead:
```
https://github.com/sample/sample/blob/6500a16c43c461ad5bb43d907a7e52c4fd13a2bf/path/to/file.m#L9
```

**Comment by calebmeyer** [comment]
I don't yet have ssh set up on my dev machine, but the fix should be pretty easy (just regex replace git@ with https). If I don't get to it tonight, I'll get to it tomorrow sometime

**Comment by calebmeyer** [comment]
Okay, @kevincantu thank you for all the bug reports. Can you test this out with the change I made?

Slack says the process to try it from master should be `apm develop git-links`

**Comment by calebmeyer** [comment]
Added 411cec7 which makes the SSH work for my github enterprise.

**Comment by calebmeyer** [comment]
Released in v0.3.0, so you don't need to run from master anymore.

**Comment by kevincantu** [comment]
:smiley_cat:
